### PR TITLE
Copyedit/revise English osu! livestreaming guide

### PR DESCRIPTION
**Pull request by bolded**
Made a proofreading and editing pass over this wiki article. Generally:
- Removed occurrences of "we" to align with the ASC
- Reduced occurrences of "you"
- Rewrote some sentences for better flow and consistency

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

SKIP_OUTDATED_CHECK still broke

### CODE REVIEW

File: wiki/Guides/Livestreaming_osu!/en.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ Now you will need to setup your account for livestreaming osu!. From the main pa
 
 ![](img/dashboard.png "Entering the Creator Dashboard")
 
-Creator Dashboard is the place where you can configure your stream information, chat moderation, profile info, and more. For now, we will focus on stream information. From the main page of the dashboard, click on the `Stream Manager` button on the left, and then choose the `Edit Stream Info` option on the right.
+In the Creator Dashboard, you can configure stream information, chat moderation, and profile info, among other customizations. Most important to getting started is configuring your stream's information. From the main page of the dashboard, click on the `Stream Manager` button on the left, and then choose the `Edit Stream Info` option on the right.
```
Comment (Walavouchey) [inline wiki/Guides/Livestreaming_osu!/en.md:13]: british english

```suggestion
In the Creator Dashboard, you can configure stream information, chat moderation, and profile info, among other customisations. Most important to getting started is configuring the stream information. From the main page of the dashboard, click on the `Stream Manager` button on the left, and then choose the `Edit Stream Info` option on the right.
```

File: wiki/Guides/Livestreaming_osu!/en.md [hunk 2]
```diff
@@ -34,9 +34,9 @@ When you launch OBS Studio for the first time, the auto-configuration wizard wil
 
 ![](img/configuration-wizard.png "OBS Studio's auto-configuration wizard")
 
-The next section to be configured are video settings. Choose the resolution of your screen for the `Base (Canvas) Resolution` field (should be detected automatically), and `Either 60 or 30, but prefer 60 when possible` for the `FPS` field.
+The next section to configure contains video settings. Choose the resolution of your screen for the `Base (Canvas) Resolution` field (should be detected automatically), and `Either 60 or 30, but prefer 60 when possible` for the `FPS` field.
```
Comment (Walavouchey) [inline wiki/Guides/Livestreaming_osu!/en.md:37]:
```suggestion
The next step is to configure some video settings. Choose the resolution of your screen for the `Base (Canvas) Resolution` field (should be detected automatically), and `Either 60 or 30, but prefer 60 when possible` for the `FPS` field.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Guides/Livestreaming_osu!/en.md (hunk 1 above)
Comment (0x84f) [reply wiki/Guides/Livestreaming_osu!/en.md:13]: _In the Creator Dashboard, you can configure stream information, chat moderation, and profile info, among other customizations._

If you are using a comma I think "and" isn't necessary?

File: wiki/Guides/Livestreaming_osu!/en.md [hunk 3]
```diff
@@ -56,9 +56,9 @@ If everything goes right, you should see the osu! window in the preview.
 
 ## What's next?
 
-Now, after you configure OBS Studio and your Twitch account, click `Start Streaming` in OBS Studio to make a livestream. Your stream will appear on [Twitch](https://www.twitch.tv/directory/game/osu!) and [osu! website's livestream listing](https://osu.ppy.sh/community/livestreams) in a couple of minutes. That's all you need to know to start your journey with streaming. Of course we have only covered the basics of livestreaming, and there are many things you can still do, with the most important of them being:
+Now, after you configure OBS Studio and your Twitch account, click `Start Streaming` in OBS Studio to make a livestream. Your stream will appear on [Twitch](https://www.twitch.tv/directory/game/osu!) and [osu! website's livestream listing](https://osu.ppy.sh/community/livestreams) in a couple of minutes. That's all you need to know to start your journey with streaming. Of course, this guide only covers the basics of livestreaming; there are many ways to go beyond and improve your livestream. So, some important points to consider:
```
Comment (0x84f) [inline wiki/Guides/Livestreaming_osu!/en.md:59]:
```suggestion
Now, after configuring OBS Studio and your Twitch account, click `Start Streaming` in OBS Studio to make a livestream. Your stream will appear on [Twitch](https://www.twitch.tv/directory/game/osu!) and [osu! website's livestream listing](https://osu.ppy.sh/community/livestreams) in a couple of minutes. That's all you need to know to start your journey with streaming. Of course, this guide only covers the basics of livestreaming; there are many ways to go beyond and improve your livestream. So, some important points to consider:
```

File: wiki/Guides/Livestreaming_osu!/en.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/Guides/Livestreaming_osu!/en.md:13]: to me it reads better

Comment (bolded) [reply wiki/Guides/Livestreaming_osu!/en.md]: Thanks, added.

File: wiki/Guides/Livestreaming_osu!/en.md (hunk 2 above)
Comment (bolded) [reply wiki/Guides/Livestreaming_osu!/en.md]: Subjective, I tried to keep it closer to the original, but your suggestion is better.

File: wiki/Guides/Livestreaming_osu!/en.md (hunk 3 above)
Comment (bolded) [reply wiki/Guides/Livestreaming_osu!/en.md]: Good idea.

**Review by bolded — COMMENTED** [review]
Excuse my awkwardness, still getting acquainted with the PR review process.

File: wiki/Guides/Livestreaming_osu!/en.md (hunk 2 above)
Comment (Walavouchey) [reply wiki/Guides/Livestreaming_osu!/en.md]: it doesn't have to be "close to the original" - if you rewrote the entire article but better that'd be fine too

**Review by Walavouchey — DISMISSED** [review]
will need to outdate translations

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**